### ISSUE TITLE
Is there performance comparison with others, such as original pytorch, DALI...

### ISSUE THREAD

**Issue by tkianai** [PhoenixDL/rising#76]
As the title

**Comment by justusschock** [comment]
Hi, what do you want benchmarken? Transforms or loading? For both we're as fast as a decent PyTorch implementation can be (since we also use PyTorch internally). We haven't yet benchmarked against DALI since they provide another featureset than we do :)

One thing to note: the transforms and the dataloader itself are comparable with plain PyTorch when it comes to speed. 
But since so far, PyTorch does not allow batches transforms (except if you do them manually afterwards, which only works without multiprocessing), the main advantage is to have these in `rising`, as they are faster then a per-sample transform.

So in general: for the same operation we're as fast as a plain PyTorch datapipeline, but we offer more possibilities when it comes to heavy augmentation workloads.

**Comment by tkianai** [comment]
Got it, nice work, thank you!
